### PR TITLE
feat(prometehus): use summary for durations and remove useless metrics

### PR DESCRIPTION
**Pull request by antoinerabany**

### CODE REVIEW

File: remoulade/middleware/prometheus.py [hunk 1]
```diff
@@ -134,39 +124,40 @@ def after_worker_shutdown(self, broker, worker):
...
+            self.message_durations
+        ]
+        for metric in metrics:
+            metric.labels(actor.queue_name, actor.options.get("prometheus_label", actor.actor_name))
+
     def after_nack(self, broker, message):
-        labels = (message.queue_name, message.actor_name)
-        self.total_rejected_messages.labels(*labels).inc()
+        labels = self._get_labels(broker, message)
+        self.total_rejected_messages.labels(labels).inc()
```
Comment (machine424) [inline remoulade/middleware/prometheus.py:141]: I think you need to unpack `labels` now that `_get_labels` returns a tuple

File: remoulade/middleware/prometheus.py [hunk 2]
```diff
@@ -134,39 +124,40 @@ def after_worker_shutdown(self, broker, worker):
...
+        # initialize the metrics for all labels to 0
+        metrics = [
+            self.worker_busy,
+            self.total_errored_messages,
+            self.total_retried_messages,
+            self.total_rejected_messages,
+            self.message_durations
+        ]
+        for metric in metrics:
+            metric.labels(actor.queue_name, actor.options.get("prometheus_label", actor.actor_name))
```
Comment (machine424) [inline remoulade/middleware/prometheus.py:137]: Ok, so labels() initializes the values, no need of set(0)?

We can remove after_worker_boot (set(0)) then.

**Review by machine424 — COMMENTED** [review]
Cool

File: remoulade/middleware/prometheus.py (hunk 2 above)
Comment (antoinerabany) [reply remoulade/middleware/prometheus.py:137]: Yeah seems more clear like this I think

**Pull request merged**